src/translations/index.md

```diff
@@ -14,6 +14,7 @@ aside: false
 
 - [Українська / Ukrainian](https://ua.vuejs.org) [[source](https://github.com/vuejs-translations/docs-ua)]
 - [Français / French](https://vuejs-docs-fr.netlify.app) [[source](https://github.com/vuejs-translations/docs-fr)]
+- [Русский / Russian](https://v3.ru.vuejs.org/) [[source](https://github.com/translation-gang/docs-ru)]
```

Comment:
Is the work in progress deployed anywhere? The URL you've suggested, <https://v3.ru.vuejs.org/>, seems to be for the old version of the docs.